alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+ENV RAILS_ENV=test                  \
+    PGBASE=/var/lib/postgresql      \
+    PGDATA=/var/lib/postgresql/data
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+ && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
```

Comment:
I wanted to avoid the "whitespace" noise when docker prints out the concatenated `RUN` statements via the line continuitation `\`. But I see value in following the convention here, will comply.